pages/index.js

```diff
@@ -22,17 +23,18 @@ function MoviesList() {
       <Heading>Movies</Heading>
       <h1>popular films</h1>
       <div className="movies">
-        {movies.map((movie) => (
-          <div key={movie.id} className="movie">
-            <h2>{movie.title}</h2>
-            <img
-              src={`https://image.tmdb.org/t/p/w500${movie.poster_path}`}
-              width="300"
-              height="300"
-              alt={movie.title}
-            />
-          </div>
-        ))}
+        {movies?.length > 0 &&
+          movies.map((movie) => (
+            <div key={movie.id} className="movie">
```

Comment:
What is the class `movie` doing? You should use `styled components` to style your html tags.

Reply:
i fix it